frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/lambda/VerifyCodeHandler.java

```diff
@@ -330,7 +330,7 @@ private void processSuccessfulCodeRequest(
         int loginFailureCount =
                 codeStorageService.getIncorrectMfaCodeAttemptsCount(session.getEmailAddress());
         var clientSession = userContext.getClientSession();
-        var clientId = userContext.getClient().get().getClientID();
+        var clientId = userContext.getClient().map(ClientRegistry::getClientID).orElse(null);
```

Comment:
What advantage does this give over simply calling getClient when the client is null?  Similar comments as previously about proceeding when the client is null.